packages/cli-e2e/utils/terminal/do.ts

```diff
@@ -70,6 +70,9 @@ export class Do implements Promise<void> {
    */
   public until(untilCondition: Condition): Promise<void> {
     let eventListener: EventListenerObject | undefined;
+    if (untilCondition instanceof RegExp) {
+      untilCondition = new RegExp(untilCondition);
+    }
```

Comment:
I need to 'clone' the regexp instance, otherwise if its used more than once, the removal of one would remove all uses.

Example above